ui/lib/pki/addon/components/pki-role-form.hbs

```diff
@@ -62,39 +57,45 @@
               data-test-toggle-group={{group}}
             />
             {{#if (get @model prop)}}
-              <div class="box is-marginless">
-                {{#let (get @model.fieldGroupsInfo group) as |groupInfo|}}
-                  {{! Header }}
-                  {{#if groupInfo.header}}
+              <div class="box is-tall is-marginless" data-test-toggle-div={{group}}>
+                {{#let (get @model.fieldGroupsInfo group) as |toggleGroup|}}
+                  {{! HEADER }}
+                  {{#if toggleGroup.header}}
                     <div class="has-bottom-margin-s">
                       <FormFieldLabel
-                        for={{groupInfo.header.name}}
-                        @label={{groupInfo.header.label}}
-                        @subText={{groupInfo.header.text}}
-                        @docLink={{groupInfo.header.docLink}}
+                        for={{toggleGroup.header.name}}
+                        @label={{toggleGroup.header.label}}
+                        @subText={{toggleGroup.header.text}}
+                        @docLink={{toggleGroup.header.docLink}}
                       />
                     </div>
                   {{/if}}
-                  {{! Fields }}
-                  {{#each fields as |attr|}}
-                    <FormField
-                      data-test-field={{true}}
-                      @attr={{attr}}
-                      @model={{@model}}
-                      @modelValidations={{@modelValidations}}
-                      @showHelpText={{false}}
-                    >
-                      {{yield attr}}
-                    </FormField>
-                  {{/each}}
-                  {{! Footer }}
-                  {{#if groupInfo.footer}}
+                  {{! FIELDS }}
+                  {{#if (eq group "Key usage")}}
+                    <PkiKeyUsage @model={{@model}} />
```

Comment:
I didn't see a need to pass anything but `@model` down? Unless I totally missed something